src/main/flight/gps_rescue.c

```diff
@@ -139,14 +147,14 @@ PG_RESET_TEMPLATE(gpsRescueConfig_t, gpsRescueConfig,
     .initialAltitudeM = 50,
     .descentDistanceM = 200,
     .rescueGroundspeed = 2000,
-    .throttleP = 150,
+    .throttleP = 60,
```

Comment:
I'm a little concerned about this default change. Have to consider that people might have tuned this value or will use a dump of previous values (or  somebody else's "working" config) and set it back to 150 or some other high value. What will be the impact if the P gain is 150 with the new logic? Might make sense to scale the value internally so that 150 externally behaves like 60. Not sure how sensitive it is to P gains or if this has been tested.

Reply:
I'm with you in this, in my local branch and in the last video I have default pids (P150, I20, D50), I'll push it after some other tests